plutus-tx-plugin/test/Budget/9.6/andWithoutGHCOpts.eval.golden

```diff
@@ -0,0 +1,2 @@
+({cpu: 1822026
+| mem: 5504})
```

Comment:
This is equal to the "local" result. Shouldn't turning off all of the GHC optimisations have some impact as well?

Reply:
The PIRs are different but after the UPLC optimizations they become the same UPLC program.

The local one doesn't run much GHC optimization either. Even though the module is compiled with optimization, the plugin is installed quite early in the core-to-core pipeline, so it runs before most optimizations.